app/containers/wallet/dialogs/WalletTrezorConnectDialogContainer.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import React, { Component } from 'react';
+import { observer, inject } from 'mobx-react';
+
+import environment from '../../../environment';
+import type { InjectedDialogContainerProps } from '../../../types/injectedPropsType';
+
+import AboutDialog from '../../../components/wallet/trezorConnect/AboutDialog';
+import ConnectDialog from '../../../components/wallet/trezorConnect/ConnectDialog';
+import SaveDialog from '../../../components/wallet/trezorConnect/SaveDialog';
+
+import { ProgressStepOption } from '../../../stores/ada/TrezorConnectStore';
+import { Logger } from '../../../utils/logging';
+
+type Props = InjectedDialogContainerProps;
+@inject('stores', 'actions') @observer
```

Comment:
pls remove the `@inject` (I got rid of them in all other files in my stores cleanup PR)

Reply:
Thanks,

likewise will remove:
https://github.com/Emurgo/yoroi-frontend/blob/4d1db2cd4983f5f006c400181d7eafea14414586/app/containers/wallet/dialogs/WalletTrezorConnectDialogContainer.js#L19

Reply:
DONE